### PR TITLE
Improve readme

### PR DESCRIPTION
**Pull request by vadi2**
#### Brief overview of PR changes/additions
Mention that gmail is OK for SMTP, mention the default user, and collapse the long steps so the README doesnt seem so overwhelming.
#### Motivation for adding to package repo
Better experience for new devs and anyone looking to get started :crossed_fingers:

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -55,6 +65,7 @@ To delete all data and restart from scratch, `docker-compose rm` and `docker vol
 Finally, start the server with run `lapis server` and visit http://localhost:8080 to see the page!
 
 The code cache is currently turned off, so refreshing the page will show any changes to the code immediately.
+</details>
 
 Submit changes via PR, and happy hacking!
```
Comment (Kebap) [inline README.md:68]: This now double with L42

**Review by Kebap — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (vadi2) [reply README.md:68]: Are you certain? This is a separate, second one that starts at L46

Comment (Kebap) [reply README.md:68]: I was talking about L70 which is after the details block.
It's the same text in L42 after the first details block.
So while minimized, you will see them double.
Maybe pull that before the details then?

Comment (vadi2) [reply README.md:68]: Sure, done.

**Pull request merged**